src/client/package.json

```diff
@@ -122,20 +123,20 @@
     "husky": "^4.3.8",
     "jest": "^27.0.0",
     "jest-styled-components": "^7.0.3",
-    "lint-staged": "^10.5.4",
+    "lint-staged": "^13.0.0",
```

Comment:
does a git hook need to be configured to run lint-staged? if I understand correctly it is not being triggered now that we've removed husky.

Reply:
I thought we had agreed that we are leaving hooks behind .. ??

Reply:
Yes, to clarify, my understanding is that lint-staged has no utility without git hooks, although perhaps I'm misunderstanding. Since I saw that we've upgraded lint-staged here, I'm looking to understand if/how it is being used as I don't see any hooks configuration.

Reply:
so if my understanding is correct, we can remove the `lint-staged` dependency and script command

Reply:
guess so, although nothing wrong with having it as a quick command to manually check before committing
but I am happy to remove it with the removal of the husky hooks if we just go with the approach of vscode + GH actions build for the branch